### PR TITLE
dotnet-format: Apply new naming rule to all projects except Vp9

### PR DESCRIPTION
**Pull request by TSRBerry**
As mentioned [here](https://github.com/Ryujinx/Ryujinx/pull/5371#discussion_r1244294177) this PR applies the new naming rule to all projects except `Ryujinx.Graphics.Nvdec.Vp9`.

This also includes the projects which still have an open formatting PR. Since all these changes were done manually it was already very tedious and I'd rather rebase them on top of this, instead of trying to separate this change into all of them.

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Controls/ApplicationContextMenu.axaml.cs [hunk 1]
```diff
@@ -10,8 +10,8 @@
 using Ryujinx.Ava.UI.ViewModels;
 using Ryujinx.Ava.UI.Windows;
 using Ryujinx.Common.Configuration;
-using Ryujinx.Ui.App.Common;
 using Ryujinx.HLE.HOS;
+using Ryujinx.Ui.App.Common;
```
Comment (TSRBerry) [inline src/Ryujinx.Ava/UI/Controls/ApplicationContextMenu.axaml.cs:14]: Unrelated change. Was done by Rider automatically since I forgot to turn off the option to optimize imports.

File: src/Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs [hunk 2]
```diff
@@ -18,14 +18,14 @@
...
 using Ryujinx.Ui.Common.Configuration;
 using Ryujinx.Ui.Common.Configuration.System;
+using Silk.NET.Vulkan;
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Runtime.InteropServices;
 using System.Net.NetworkInformation;
+using System.Runtime.InteropServices;
```
Comment (TSRBerry) [inline src/Ryujinx.Ava/UI/ViewModels/SettingsViewModel.cs:27]: Same thing here.

File: src/Ryujinx.Graphics.Gpu/Memory/PhysicalMemory.cs [hunk 3]
```diff
@@ -6,8 +6,8 @@
 using Ryujinx.Memory.Tracking;
 using System;
 using System.Collections.Generic;
-using System.Runtime.InteropServices;
 using System.Linq;
+using System.Runtime.InteropServices;
```
Comment (TSRBerry) [inline src/Ryujinx.Graphics.Gpu/Memory/PhysicalMemory.cs:10]: Another one.

File: src/Ryujinx.Graphics.Vulkan/BufferManager.cs [hunk 4]
```diff
@@ -4,8 +4,8 @@
 using System;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-using VkFormat = Silk.NET.Vulkan.Format;
 using VkBuffer = Silk.NET.Vulkan.Buffer;
+using VkFormat = Silk.NET.Vulkan.Format;
```
Comment (TSRBerry) [inline src/Ryujinx.Graphics.Vulkan/BufferManager.cs:8]: This one as well.

File: src/Ryujinx.Ui.Common/App/ApplicationLibrary.cs [hunk 5]
```diff
@@ -18,9 +18,9 @@
 using Ryujinx.Ui.Common.Configuration.System;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Globalization;
 using System.IO;
+using System.Linq;
```
Comment (TSRBerry) [inline src/Ryujinx.Ui.Common/App/ApplicationLibrary.cs:23]: Hopefully the last one.

File: src/Ryujinx.Ui.Common/App/ApplicationData.cs [hunk 6]
```diff
@@ -1,9 +1,9 @@
 ﻿using LibHac.Common;
-using LibHac.Ns;
 using LibHac.Fs;
 using LibHac.Fs.Fsa;
 using LibHac.FsSystem;
 using LibHac.Loader;
+using LibHac.Ns;
```
Comment (TSRBerry) [inline src/Ryujinx.Ui.Common/App/ApplicationData.cs:6]: Missed that one.

File: src/Ryujinx.Tests/Cpu/CpuTestSimdMemory32.cs [hunk 7]
```diff
@@ -1,8 +1,8 @@
 ﻿#define SimdMemory32
 
 using ARMeilleure.State;
-using Ryujinx.Memory;
 using NUnit.Framework;
+using Ryujinx.Memory;
```
Comment (TSRBerry) [inline src/Ryujinx.Tests/Cpu/CpuTestSimdMemory32.cs:5]: Also this one.

**Review by TSRBerry — COMMENTED** [review]
I hope it's okay to keep them, since these changes would have been done in the other formatting PRs as well.

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Review by AcK77 — APPROVED** [review]
